qgis-projects/README.md

```diff
@@ -0,0 +1,141 @@
+- Source: Connect to the database, create a local geopackage named dominode_data.gpkg and select all features, then drag and drop them onto the gpkg within the QGIS Browser. Contact the custodian for database connection details.
+ 
+---
+
+- File: ./data/auxillary_data.gpkg
+- Custodian: Default
+- Description: Additional data required but not present in core database. This data should be accounted for in the remote data source later on.
+- Source: [Download from google drive](https://drive.google.com/file/d/1lH3dr4Ic3EZmHfmmZXPCjM-vbp6V6epN/view?usp=sharing)
+
+---
+
+- File: ./local/COMPASS_ROSE.geojson
+- Custodian: Default
+- Description: Feature generated with the Magnetic Declination QGIS plugin
+- Source: Contact custodian
+
+---
+
+> Note that the *data* directory is intended to be a symbolic link to a local data directory separate from the project repository.
+
+## File locks
+
+To prevent unwanted/ implicit changes to map projects, custodians may modify the file permissions for map projects to "read-only" prior to performing a commit.
+
+## Directory structure
+
+Primary custodians for the qgis-projects directory should be indicated in the ```CODEOWNERS``` in the *Project Root*.
+
+```text
+qgis-projects/    @user01 @user02
+```
+
+Additional ownership can be added to the ```CODEOWNERS``` file where relevant.
+
+```text
+qgis-projects/file-name.qgs    @user03
+```
+
+The following lines should be added to ```.gitignore``` in the *Project Root*.
+
+```text
+qgis-projects/data
+qgis-projects/**/*.qgs~
+qgis-projects/**/*.qgz~
+```
+
+This prevents git from interfering with the symbolic link to the data directory, which is likely to differ between users and operating systems. It also prevents the inclusion of QGIS autogenerated "backup" files, which are largely unnecessary when utilising a distributed VCS such as git.
+
+The following lines should be added to ```.gitattributes``` in the *Project Root*.
+
+```text
+qgis-projects/**/*.qgs lockable
+qgis-projects/**/*.qgz lockable
+```
+
+This ensures that QGIS project files are made available for *locking* by the Git LFS system.
+
+There are a number of directories utilised within a qis-projects directory (note that not all directories may be present until the relevant data is available). Common directories, their naming conventions and their functions are outlined by the following list.
+
+- README.md: This readme document in markdown format.
+- assets: Multimedia assets directory. This may include images used in this readme document, corporate stationary objects or assets for use in the cartographic process, such as svg libraries.
+- atlas: Default output directory for atlas items. Storing this directory within the project repository should be used with caution, as atlas outputs are often large binary data.
+- data: Symbolic link to large file storage
+- docs: Additional documents and addenda
```

Comment:
Let's keep an optional `docs` directory at the repo root for storing any documentation about the repo, if necessary instead of this